ci/jobs/development.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- name: development
+  plan:
+  - get: dataworks-analytical-service-infra
+    trigger: true
+  - .: (( inject meta.plan.terraform-bootstrap ))
+    params:
+      DEPLOY_PATH: infra
+  - .: (( inject meta.plan.terraform-apply ))
+    config:
+      run:
+        dir: 'dataworks-analytical-service-infra/terraform/deploy/infra'
+  - .: (( inject meta.plan.terraform-plan ))
+    config:
+      run:
+        dir: 'dataworks-analytical-service-infra/terraform/deploy/infra'
```

Comment:
this should be in the meta file, it doesn't vary per job

Reply:
done - moved to meta.yml